init/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# This Dockerfile is only used to init your dib managed repository
+#
+# Within your dib managed repository, copy this Dockerfile and run the
+# following commands :
+# $ docker build -t <your_registry_url>/dib-referentiel .
+# $ docker push <your_registry_url>/dib-referentiel
+#
+# If you choose to use another name than `dib-referentiel`, you will need
+# to set it in your command-line, with `--referentiel-image`
```

Comment:
Please ensure it matches the name of the flag once you've fixed my previous comments